src/visual.ts

```diff
@@ -458,11 +470,18 @@ module powerbi.extensibility.visual {
 
             tooltipItems.forEach((tooltipItem: BulletChartTooltipItem) => {
                 if (tooltipItem && tooltipItem.metadata) {
-                    let metadata: DataViewMetadataColumn = tooltipItem.metadata.source,
+                    let displayName: string,
+                        metadata: DataViewMetadataColumn = tooltipItem.metadata.source,
                         formatString: string = valueFormatter.getFormatStringByColumn(metadata);
 
+                    if (tooltipItem.customName) {
+                        displayName = tooltipItem.customName;
+                    } else {
+                        displayName = metadata.displayName;
+                    }
+
                     tooltipDataItems.push({
-                        displayName: metadata.displayName,
+                        displayName: displayName,
```

Comment:
You might use `displayName,` instead of `displayName: displayName,`. These constructions are the same.